dotCMS/src/main/java/com/dotcms/rest/api/v1/contenttype/ContentTypeResource.java

```diff
@@ -106,8 +115,9 @@ public final Response createType(@Context final HttpServletRequest req, final Co
 						.putAll(new JsonContentTypeTransformer(contentTypeSaved).mapObject())
 						.put("workflows", this.workflowHelper.findSchemesByContentType(contentTypeSaved.id(), initData.getUser()))
 						.build();
-
 				retTypes.add(responseMap);
+				// save the latest to the session to be compliant with #13719
+				req.getSession().setAttribute("selectedStructure",contentTypeSaved.inode());
```

Comment:
@wezell getSession(false) to avoid creating a new session? and validate not null?

Reply:
Good eye Daniel.  Yes to both.

Reply:
As far as I can tell there is always a session. Even when the request comes from POST Man or curl.  Probably it gets created by an authentication filter or something.    
The additional checks were added. 

Also added a Unit test that validates the existence of the attribute.